packages/documentation-v7/src/stories/home-tile.components.ts

```diff
@@ -0,0 +1,30 @@
+import { LitElement, html, unsafeCSS } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import styles from './home-tile.components.scss?inline';
+
+@customElement('home-tile')
+export class HomeTile extends LitElement {
+    static styles = unsafeCSS(styles);
+
+    @property() title: string = '';
+    @property() href?: string;
+
+    render() {
+      return this.href
+        ? html`
+          <a class="tile" href=${this.href} target="_blank" rel="noopener">
+              ${ this.title ? html`<span class="tile--title">${ this.title }</span>` : null }
+              <span class="tile--icon">
+              <slot></slot>
+              </span>
+          </a>`
+        : html`
+          <div class="tile">
+              ${ this.title ? html`<span class="tile--title">${ this.title }</span>` : null }
+              <span class="tile--icon">
+              <slot></slot>
+              </span>
+          </div>`

```

Comment:
[Static expressions](https://lit.dev/docs/templates/expressions/#static-expressions) could be potentially used to avoid duplication here, but I ran into the following issue while trying to set an attribute (e.g. `href`) as an expression: "String contains an invalid character".